### ISSUE TITLE
pull the libgfortran dependency in the mxnet pip package

### ISSUE THREAD

**Issue by nswamy** [apache/incubator-mxnet#8903]
## Description
I installed mxnet and as soon as I imported mxnet, it failed with this ugly error: Please add all the dependencies of mxnet to pip package.
```
ubuntu@ip-172-30-0-224:~$ python3
Python 3.5.2 (default, Nov 23 2017, 16:37:01)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mxnet as mx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ubuntu/.local/lib/python3.5/site-packages/mxnet/__init__.py", line 25, in <module>
    from .base import MXNetError
  File "/home/ubuntu/.local/lib/python3.5/site-packages/mxnet/base.py", line 111, in <module>
    _LIB = _load_lib()
  File "/home/ubuntu/.local/lib/python3.5/site-packages/mxnet/base.py", line 103, in _load_lib
    lib = ctypes.CDLL(lib_path[0], ctypes.RTLD_LOCAL)
  File "/usr/lib/python3.5/ctypes/__init__.py", line 347, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libgfortran.so.3: cannot open shared object file: No such file or directory
## Environment info (Required)

```
What to do:
pip3 Install mxnet 
python3
import mxnet as mx

**Comment by nswamy** [comment]
On ubuntu, 'sudo apt-get install gfortran' works.

**Comment by szha** [comment]
pip doesn't manage dependencies for c-libraries so it's couldn't just be added as you suggested. For other c-libraries I usually just statically link them. However, libgfortran can't be statically linked without building it from gcc. Building it from gcc doesn't quite fit in the time limits of the current build system.

@nswamy in short-term if you have time please submit a patch to give warning on how to get libgfortran to make the error less ugly.

**Comment by szha** [comment]
Oops wrong button.

**Comment by szha** [comment]
The problem should have been resolved in mxnet==1.0.0.post2, pending validation. This issue has the same resolution as 9048 so let's track the progress there.

**Comment by atodkar** [comment]
@nswamy, worked your solution. @szha, well i have installed 1.0.0.post2 got above issue.
